scripts/fuzz-parser/fuzz.py

```diff
@@ -244,21 +269,28 @@ def parse_args() -> ResolvedCliArgs:
     parser.add_argument(
         "--test-executable",
         help=(
-            "`ruff` executable to test. "
+            "Executable to test. "
             "Defaults to a fresh build of the currently checked-out branch."
         ),
     )
     parser.add_argument(
         "--baseline-executable",
         help=(
-            "`ruff` executable to compare results against. "
-            "Defaults to whatever `ruff` version is installed "
+            "Executable to compare results against. "
+            "Defaults to whatever version is installed "
             "in the Python environment."
         ),
     )
+    parser.add_argument(
+        "--bin",
+        help=("Name of executable to test. E.g. `ruff` or `red_knot`."),
+        required=True,
+    )
```

Comment:
Can we limit this to just "ruff" or "red_knot" using the `choices` field? (https://docs.python.org/3/library/argparse.html#choices)